New postscript (#870)
browserengineering/book: book/change.md

prev: skipped
...

The web is a dynamic, ever-changing place. The first web browser, in
1989, did not support colors, images, styling, or scripting. Three
decades of evolution driven by market forces, implementation quirks,
and the ever-expanding reach of the web then made browsers what they

> are today. Those forces are stronger than ever today.

Reviewer: ```suggestion
are today. And those forces are as strong as ever, so there's every reason to expect
this to continue in the future.
```

Author: Hmm, this seems overly wordy. How about

> Those forces as as strong as ever. Browsers will continue to evolve.

Sooner or later this book will be obsolete---the sooner the better!

> Whether it is Web Assembly or WebGPU, hardware access or new CSS

Reviewer: ```suggestion
Whether it is WebAssembly or WebGPU, hardware access or new CSS
```

features, integrated payments or AI assistants, I do expect the

> browser of the future to play a totally different role in our

Reviewer: ```suggestion
browser of the future to play many new and different roles in 
```
